src/main/java/de/rabea/ui/PrettyBoardPainter.java

```diff
@@ -7,26 +7,26 @@
 import static de.rabea.game.Mark.X;
 
 public class PrettyBoardPainter implements BoardPainter {
-    private final String BLUE_COLOUR_FOR_X = "\u001B[34m";
-    private final String RED_COLOUR_FOR_O = "\u001B[31m";
+    private final String BLUE_COLOUR_FOR_X = " \u001B[34m";
+    private final String RED_COLOUR_FOR_O = " \u001B[31m";
     private final String COLOUR_RESET = "\u001B[0m";
 
     @Override
     public String drawBoard(Board board) {
-        int i = 0;
+        int positionNumberToPrint = 0;
         String boardImage = "\n";
         for (Mark cell : board.cells()) {
-            i ++;
-            boardImage = printSymbolInCell(i, boardImage, cell);
-            boardImage = printHorizontalLines(i, boardImage, board);
-            boardImage = printLastPipe(i, boardImage, board);
+            positionNumberToPrint ++;
+            boardImage = printSymbolInCell(positionNumberToPrint, boardImage, cell);
+            boardImage = printHorizontalLines(positionNumberToPrint, boardImage, board);
+            boardImage = printLastPipe(positionNumberToPrint, boardImage, board);
         }
         return boardImage;
     }
 
-    private String printSymbolInCell(int i, String boardImage, Mark cell) {
+    private String printSymbolInCell(int positionNumberToPrint, String boardImage, Mark cell) {
         if (cell == EMPTY) {
-            boardImage += pipe() + i;
+            boardImage += pipe() + String.format("%2d", positionNumberToPrint);
```

Comment:
What does this `Sring.format(...)`? Extract it to a method and give a decent name.